examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/templates/partials/command_response_type.zapt

```diff
@@ -0,0 +1,19 @@
+{{#if command.hasSpecificResponse}}
+{{#*inline "clusterName"}}
+{{~#if compatRemapNames~}}
```

Comment:
I really doubt you need any of this compat name mapping business.

Reply:
Same. It was just copy/pasted from Matter.framework. Removing.